mmap_unix.go missing BSD in Go build tag

https://github.com/bytedance/sonic/blob/8c71eb047516a6b70f216887923f353f42d0501b/loader/mmap_unix.go#L1C1-L2C23

Would it be possible to make the _loader/mmap_unix.go_ file target more Unix based OSes than just Darwin and Linux? I'm trying to build a project that indirectly depends on this package and I get the following build error.

```
# github.com/bytedance/sonic/loader
../../go/pkg/mod/github.com/bytedance/sonic@v1.10.2/loader/funcdata_latest.go:228:13: undefined: mmap
../../go/pkg/mod/github.com/bytedance/sonic@v1.10.2/loader/funcdata_latest.go:233:5: undefined: mprotect
```

--- a/loader/mmap_unix.go
+++ b/loader/mmap_unix.go
@@ -1,5 +1,5 @@
-//go:build darwin || linux
-// +build darwin linux
+//go:build unix
+// +build unix
 
 /**
  * Copyright 2023 ByteDance Inc.

--- a/loader/mmap_unix.go
+++ b/loader/mmap_unix.go
@@ -1,5 +1,5 @@
-//go:build unix
-// +build unix
+//go:build !windows
+// +build !windows
 
 /**
  * Copyright 2023 ByteDance Inc.

